### PR TITLE
Adding Raptor dashboards

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

File: grafana/app/dashboards/mark.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  refresh: false,
+  version: 6,
+  hideAllLegends: false
+};
+var basePanel = {
+  error: false,
+  span: 4,
+  editable: false,
+  type: 'graph',
+  datasource: 'raptor',
```
Comment (eliperelman) [inline grafana/app/dashboards/mark.js:79]: Does this datasource need to be changed for the current stats site?

Comment (jonasfj) [inline grafana/app/dashboards/mark.js:79]: It's it just the name of the data source in grafana:
https://github.com/taskcluster/taskcluster-stats/blob/master/grafana/config.js#L54

So think this is okay...

**Comment by jonasfj** [comment]
Looks right to me... I assume you placed the data file in the right folders... Which should make it all work.

**Comment by jonasfj** [comment]
I'll push this to heroku...

**Pull request merged**